src/sentry/api/endpoints/notifications/notification_actions_index.py

```diff
@@ -0,0 +1,46 @@
+from typing import Dict
+
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization_flag import FlaggedOrganizationEndpoint
+from sentry.api.serializers.base import serialize
+from sentry.api.serializers.rest_framework.notification_action import NotificationActionSerializer
+from sentry.models.notificationaction import NotificationAction
+from sentry.models.organization import Organization
+
+
+@region_silo_endpoint
+class NotificationActionsIndexEndpoint(FlaggedOrganizationEndpoint):
+    feature_flags = ["organizations:notification-actions"]
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        queryset = NotificationAction.objects.filter(organization_id=organization.id)
+
+        project_id_query = request.GET.getlist("projectId")
```

Comment:
Generally project filter parameters are called `project`. This is what `OrganizationEndpoint` uses to filter by project.

Reply:
That's actually great, I used the `get_projects` helper as well thanks :+1: